Tactics/CSE.lean

```diff
@@ -104,6 +104,13 @@ structure CSEConfig where
   minOccsToCSE : Nat := 2
   /-- Whether the tactic should throw an error if no CSEable subterms were found. -/
   failIfUnchanged : Bool := true
+  /-- Number of steps the tactic should spend searching for subterms to gather information. -/
```

Comment:
What is the motivation for having two separate fuels `searchFuel` and `eliminateFuel`? Could they be represented as one fuel?

Reply:
I added some text for why it's useful to have separate fuels. Conceptually, the algorithm has two phases, and it's useful for both debugging and for controlling behaviour to be able to fuel these separately.